0.0.6 install.sh fails

Expected `install.sh` to start prompting me but instead it fails right out of the gate

To reproduce:
```
git clone https://github.com/databrickslabs/ucx

ucx % export DATABRICKS_CONFIG_PROFILE=HLSFIELD
ucx % bash ./install.sh 
[i] found Python 3.9.6 -> /usr/bin/python3
[i] found Python 3.10.10 -> /Users/douglas.moore/.pyenv/shims/python3
[i] latest python is /Users/douglas.moore/.pyenv/shims/python3
[i] installing venv into: /var/folders/d6/5qdmg6pn5kgbnnjcgpg0lxwr0000gp/T/tmp.SYLQ2ywr
[+] making sure we have the latest pip version
[+] installing dependencies within ephemeral Virtualenv: /var/folders/d6/5qdmg6pn5kgbnnjcgpg0lxwr0000gp/T/tmp.SYLQ2ywr
Traceback (most recent call last):
  File "/Users/douglas.moore/.pyenv/versions/3.10.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/douglas.moore/.pyenv/versions/3.10.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/douglas.moore/development/ucx/src/databricks/labs/ucx/install.py", line 908, in <module>
    installer.run()
  File "/Users/douglas.moore/development/ucx/src/databricks/labs/ucx/install.py", line 149, in run
    logger.info(f"Installing UCX v{self._version}")
  File "/Users/douglas.moore/development/ucx/src/databricks/labs/ucx/install.py", line 795, in _version
    raise OSError(msg) from None
OSError: Cannot determine unreleased version. Please report this error message that you see on https://github.com/databrickslabs/ucx/issues/new. Meanwhile, download, unpack, and install the latest released version from https://github.com/databrickslabs/ucx/releases. Original error is: 'NoneType' object has no attribute 'split'
```

Also for `make test`

```================================= short test summary info ==================================
FAILED tests/unit/test_install.py::test_build_wheel - OSError: Cannot determine unreleased version. Please report this error message that you...
FAILED tests/unit/test_install.py::test_main_with_existing_conf_does_not_recreate_config - OSError: Cannot determine unreleased version. Please report this error message that you...
FAILED tests/unit/test_install.py::test_replace_clusters_for_integration_tests - OSError: Cannot determine unreleased version. Please report this error message that you...
============================== 3 failed, 247 passed in 5.29s ===============================
```

--- a/src/databricks/labs/ucx/install.py
+++ b/src/databricks/labs/ucx/install.py
@@ -776,7 +776,7 @@ class WorkspaceInstaller:
             dv = SemVer.parse(git_detached_version)
             datestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
             # new commits on main branch since the last tag
-            new_commits = dv.pre_release.split("-")[0]
+            new_commits = dv.pre_release.split("-")[0] if dv.pre_release else None
             # show that it's a version different from the released one in stats
             bump_patch = dv.patch + 1
             # create something that is both https://semver.org and https://peps.python.org/pep-0440/

